### DISCUSSION TITLE
ある方のQ&A（質問）に回答しましたが、マイページの「回答した質問」にありません

### DISCUSSION THREAD

**Discussion by nak435** [increments/qiita-discussions#540] [ヘルプ]
ある方のQ&A（質問）に回答しましたが、マイページの「回答した質問」にありません。

つい最近の質問でしたので、質問タグの**すべて**で見たところ、どうも質問自体が無くなっています。
おそらくQ&Aされた方が、ご自身で**削除**されたのかなと想定します。

そこで、確認ですが、投稿した質問を質問者さんが削除すると、それに答えた側のマイページの「回答した質問」からも消える仕様でしょうか？
（回答した履歴としては、削除済みでも一覧に残りして欲しいなと思いました）

同様に、記事自体が削除されると、その記事にコメントした履歴も消えてしまいますか？

**Comment by tsubolabo** [comment]
@nak435 

若しかすると、私の質問のことでしょうか？

確かに削除しました。その**理由は同一内容で再質問するため**です。

同一内容の質問が重複しては迷惑かと思い、旧質問は削除しました。少しでも役に立つコメントが付いている質問であれば、削除するのは気が引けますが、コメント内容は「ここではなく、他で聞いた方が良い」とういうような内容でしたので、失礼ながら、あまり気にすることもなく削除してしまいました。申し訳ありません。

**再質問した理由は、[再質問](https://qiita.com/tsubolabo/questions/9d4a9463937df029f5bd)の冒頭に書いてあるとおり**です。

ここのdiscussionの「**質問が表示される期間が短い** #527」で指摘されている問題を回避するためでした。しかし、一週間ごとに繰り返すのも大変です。１回限りで止めにします。

**Reply by nak435** [reply]
> 若しかすると、私の質問のことでしょうか？

いいえ、@tsubolabo さんではありません。VBAを質問された方です。

**Comment by tanaka-asami415** [comment]
@nak435 
> そこで、確認ですが、投稿した質問を質問者さんが削除すると、それに答えた側のマイページの「回答した質問」からも消える仕様でしょうか？
（回答した履歴としては、削除済みでも一覧に残りして欲しいなと思いました）
>
> 同様に、記事自体が削除されると、その記事にコメントした履歴も消えてしまいますか？

はい。恐れ入りますが、現在はご指摘の通り質問や記事が削除されると、回答や記事へのコメントも見えなくなってしまう仕様となっております🙇

同様のご意見もいただいているため、改善を検討しております🙇
https://github.com/increments/qiita-discussions/discussions/230

**Reply by nak435** [reply]
追加の確認です。
もし、回答やコメントに「いいね」をもらっていたとして、その質問や記事が削除されると、「いいね」カウントも減りますか？

**Reply by tanaka-asami415** [reply]
@nak435 
現在の仕様では「いいね」のカウント及びcontribution数も減ってしまう仕様となっております。
回答やコメントを残すようにすれば、加算されたままにできるので合わせて検討いたします🙇

**Reply by nak435** [reply]
現在の仕様については理解しました。
改善していただけるということで、よろしくお願いします。

**Reply by tanaka-asami415** [reply]
@nak435 
お待たせいたしました🙇

まずは回答がついた質問は削除できないようにいたしました！
https://qiita.com/release-notes/2023-10-04

まだユーザーが退会してしまった場合は消えてしまう状態なのですが、こちらも追って対応を予定しております。
コメントを残すようにする変更については、影響範囲が大きいため別途対応を検討しています。

今後の修正は #230  でご報告いたしますので、本Discussionはクローズさせていただきます🙇
